lib/screens/add_account_flow.dart

```diff
@@ -281,6 +279,7 @@ class AddAccountFlowState extends FlowBaseState<AddAccountFlow>
     );
 
     if (link != null) {
+      String _multiSigInvalidLink = Strings.of(context).multiSigInvalidLink;
```

Comment:
Could remove the underscore to keep with convention and also make this `final`, or remove the local variable altogether.